armi/reactor/grids.py

```diff
@@ -436,6 +436,11 @@ def __iter__(self):
     def __len__(self):
         return len(self._locations)
 
+    def detachedCopy(self):
+        loc = MultiIndexLocation(None)
+        loc.extend(self._locations)
+        return loc
```

Comment:
probably needs explanatory docstring?

Reply:
Nothing really to be gained from what's in the base class's docstring